test/runtests.jl

```diff
@@ -10,11 +12,12 @@ tests = [
     "DataOneV2/TERN",
     "CKAN",
     "DataCite",
-    "format_checksum"
 ]
 
-for filename in tests
-    @testset ExtendedTestSet "$filename" begin
-        include(filename * ".jl")
+@testset "DataDepGenerators" begin
```

Comment:
This suppresses the individual tests. I am not sure if that is wanted?

Reply:
It vainly surpresses them if they pass